VisualStudio.gitignore

```diff
@@ -51,6 +51,13 @@ project.fragment.lock.json
 artifacts/
 **/Properties/launchSettings.json
 
+# StyleCop
+StyleCopReport.xml
+
+# Thumbnails created by windows
+Thumbs.db
```

Comment:
This is covered by `Global/Windows.gitignore` - please remove it.

Reply:
Removed